.github/workflows/retry-tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Retry Tests
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  test-momento-local-1:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Start Momento Local 1
+        run: |
+          docker run -d -p 8080:8080 gomomento/momento-local --return-error unavailable --error-rpcs get
```

Comment:
Once the request-based momento-local is available, we can just have one service that the tests can use right?